_pages/index.html

```diff
@@ -10,13 +10,14 @@
     <!-- Container 1  (5 protein morning snacks) -->
     <div> 
         <h1>
-            <strong>
-            15 Protein-Packed Meals That Are 
+           <p class="Title">
+         15 Protein-Packed Meals That Are 
             <em>Easy
             </em>to Make!
-        </strong>
+        </p>
+       
         </h1>
-        <img src="/Users/safiya/Desktop/protein meal.jpeg" width="450" height="250">
+        <img src="/Users/safiya/Desktop/protein meal.jpeg" class="Mealprep">
        <br>
         <h2>
         <p class="breakfast-easy-fix"> 
```

Comment:
This classname is good an your others should follow its format and be consistent

Reply:
thank you